### DISCUSSION TITLE
Northeastern US growth & mortality use case

### DISCUSSION THREAD

**Discussion by Kah5** [mekevans/forestTIME#22] [General]
Starting a discussion for the use case of tracking individual tree growth rates & mortalities over multiple remeasurements throughout Northeastern US plots. Right now the focus is on several northeastern states where tree cores were previously sampled, but we may extend it out later. States include OH, PA, MD, MA, NJ, VT, NH, RI, ME, CT, WV, and NY. We may want to extend to the rest of the Northern Research station R9, which includes IN, IA, IL, MI, MO, WI, and MN.

**Comment by diazrenata** [comment]
Hi @Kah5! Here's a box link to the variables we talked about earlier this week, for all of these states: https://arizona.box.com/s/4efq3jt0vm3ezzj68ahckcftb9zb53ym

Here's also a document with instructions + a demo for how you (hopefully!) can query things on your own: https://github.com/mekevans/forestTIME/blob/main/use_cases/tree_rings/TreeRing.md It includes downloading a database (1.5GB) of the TREE, PLOT, and CONDITION tables + the persistent tree CN tables. I generated this on my computer from the version of the data I could download from DataMart. We might need to work together a bit to get it set up so you can create it independently starting from the version of FIADB you're using - there are some scripts, but the first step (getting the TREE, PLOT, and CONDITION tables) might need some figuring-out.

**Comment by Kah5** [comment]
Thanks Renata! I was able to access the box file, but it looks like its
just for whitebark pine.

I took a look at the demo and it all looks pretty thorough to me (though I
will get back to you when I try it out)!

**Comment by diazrenata** [comment]
Hi Kelly,

Oh no, I’m sorry! That must have been the link for Ceci. Try this one? https://arizona.box.com/s/nlykl9rbchlk2bj9npjd8dej5iw06i8a

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: Kelly Heilman ***@***.***>
Sent: Wednesday, February 14, 2024 4:45:35 PM
To: mekevans/forestTIME ***@***.***>
Cc: Diaz, Renata - (renatadiaz) ***@***.***>; Comment ***@***.***>
Subject: [EXT]Re: [mekevans/forestTIME] Northeastern US growth & mortality use case (Discussion #22)


External Email

Thanks Renata! I was able to access the box file, but it looks like its
just for whitebark pine.

I took a look at the demo and it all looks pretty thorough to me (though I
will get back to you when I try it out)!

**Comment by Kah5** [comment]
Okay that one looks like it is a .duckdb extension! I will test this out,
probably next week. Thank you!
